frontend/src/components/Home.vue

```diff
@@ -149,7 +174,76 @@ export default {
       } else {
         return `$${price.toFixed(2)}`
       }
+    },
+    formatPricePadding(price) {
+      function pad(n, width, z) {
+        z = z || "0"
+        n = n + ""
+        return n.length >= width ? n : new Array(width - n.length + 1).join(z) + n
+      }
+      if(price == null) {
+        return "------"
+      } else { // dirtyhack
+        return `$${pad(price.toFixed(2), 8, "\xa0")}`
+      }
+    },sortTable() {
+        var table, rows, switching, i, x, y, shouldSwitch
+        table = document.getElementById("tbl2")
+        switching = true
+        while (switching) {
+          switching = false
+          rows = table.getElementsByTagName("TR")
+          for (i = 1; i < (rows.length - 1); i++) {
+            shouldSwitch = false
+            x = rows[i].getElementsByTagName("TD")[0]
+            y = rows[i + 1].getElementsByTagName("TD")[0]
+            if(x.innerHTML.toLowerCase() > y.innerHTML.toLowerCase()) {
+              shouldSwitch = true
+              break
+            }
+          }
+          if(shouldSwitch) {
+            rows[i].parentNode.insertBefore(rows[i + 1], rows[i])
+            switching = true
+          }
+        }
+      },
+    sortBy: function(sortKey) {
```

Comment:
These should be defined like `sortBy(sortKey) {` to match existing code.

Also, try to keep consistent indentation